master/buildbot/steps/trigger.py

```diff
@@ -198,13 +210,21 @@ def addBuildUrls(self, rclist):
     @defer.inlineCallbacks
     def run(self):
         schedulers_and_props = yield self.getSchedulersAndProperties()
+        
+        # To be back compatible we need to differ between old and new style
+        # schedulers_and_props can either consist of 2 elements or 3 elements tuples
+        
+        # If the user getSchedulersAndProperties funcion does not specify the importance (red len(tup)==2), we make all schedulers important
+        schedulers_and_props = [tup if len(tup)==3 else (tup[0], tup[1], False) for tup in schedulers_and_props]
```

Comment:
I think it would be better to change to a dict or better, a namedtuple so that futureproof is even easier.
https://docs.python.org/3/library/collections.html#collections.namedtuple

If we need to add another argument we would need if len == 4, which will be even worse.
I usually dont like too much if-in-the-future arguments, but in this case this choice wont be undoable as this is API.

Another option is to add self.getSchedulersAndPropertiesWithImportance, which would call the older version, but this is not very clean either.